.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pycmake"]
```

Comment:
Is this just a preparation, or does this PR actually introduce a dependency on pycmake.